File: WeatherApp_Obj-C/WeatherEntity+CoreDataProperties.m
//
//  WeatherEntity+CoreDataProperties.m
//  WeatherApp_Obj-C
//
//  Created by Muradasil Birhan on 05/03/2020.
//  Copyright © 2020 RodeApps. All rights reserved.
//
//

#import "WeatherEntity+CoreDataProperties.h"

@implementation WeatherEntity (CoreDataProperties)

+ (NSFetchRequest<WeatherEntity *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"WeatherEntity"];
}

@dynamic city;
@dynamic imgUrl;
@dynamic lat;
@dynamic lon;
@dynamic mainEvent;
@dynamic name;
@dynamic rain;
@dynamic region;
@dynamic temperature;
@dynamic time;
@dynamic wind;
@dynamic user;

@end

File: WeatherApp_Obj-C/WeatherEntity+CoreDataProperties.swift
//
//  WeatherEntity+CoreDataProperties.swift
//  
//
//  Created by Muradasil Birhan on 02/03/2020.
//
//

import Foundation
import CoreData


extension WeatherEntity {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<WeatherEntity> {
        return NSFetchRequest<WeatherEntity>(entityName: "WeatherEntity")
    }

    @NSManaged public var city: String?
    @NSManaged public var imgUrl: String?
    @NSManaged public var lat: Double
    @NSManaged public var lon: Double
    @NSManaged public var mainEvent: String?
    @NSManaged public var name: String?
    @NSManaged public var rain: Double
    @NSManaged public var region: String?
    @NSManaged public var temperature: Double
    @NSManaged public var time: Int32
    @NSManaged public var wind: Double

}
